ui/src/session.ts

```diff
@@ -48,10 +47,16 @@ sessionStore.subscribe(data => {
 });
 
 // Return the unseleased session if the session is unsealed, undefined otherwise.
-export const unsealed = (): UnsealedSession | undefined => {
+export const unsealed = (): UnsealedSession => {
```

Comment:
Could you update the documentation of this function?

Reply:
0fff2cefd7959ff357eb8cc0a3f10eee99aa5532